src/Sylius/Bundle/ApiBundle/DataTransformer/LoggedInCustomerEmailAwareCommandDataTransformer.php

```diff
@@ -27,13 +26,16 @@ public function __construct(private UserContextInterface $userContext)
     {
     }
 
-    public function transform($object, string $to, array $context = [])
+    /**
+     * @param mixed $object
+     * @param array<string, mixed> $context
+     */
+    public function transform($object, string $to, array $context = []): object
     {
-        /** @var CustomerInterface|null $customer */
         $customer = $this->getCustomer();
 
         /** @var CustomerEmailAwareInterface|mixed $object */
-        Assert::object($object, CustomerEmailAwareInterface::class);
+        Assert::isInstanceOf($object, CustomerEmailAwareInterface::class);
```

Comment:
Similar case, we should either type it as `object` or check that it's object in the first place